src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -337,34 +296,26 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
                 BankAccounts.hideBankAccountErrors();
             }
 
-            // eslint-disable-next-line no-shadow
-            const policyID = lodashGet(route.params, 'policyID');
-            const backTo = lodashGet(route.params, 'backTo');
+            // eslint-disable-next-line no-shadow, @typescript-eslint/no-shadow
+            const policyID = route.params.policyID;
```

Comment:
Maybe we could just rename `policyID` to get rid of the warnings and use the renamed variable below